docs/user-guide/deployments.md

```diff
@@ -454,6 +454,142 @@ nginx-deployment-3066724191   0         0         1h
 Note: You cannot rollback a paused Deployment until you resume it.
 
 
+## Failed Deployment
+
+Your Deployment may get stuck trying to deploy its newest ReplicaSet without ever completing, due to:
+
+* insufficient quota
+
+* readiness probe failures
+
+* image pull error
+
+* insufficient permissions
+
+* limit ranges
+
+* application runtime misconfiguration
+
+In those cases you can specify a timeout parameter in the spec ([spec.progressDeadlineSeconds](#progress-deadline-seconds))
+that denotes the seconds you want to wait for your Deployment to report any progress. Once the deadline is exceeded, meaning
```

Comment:
Also, put in a paragraph break after "progress"; you need to insert the text from line 509 afterward.